source/includes/_products.md

```diff
@@ -1 +1,255 @@
 # Products
+
+## Product Object
+
+Attribute | Type | Description
+--------- | ---- | -------
+active | `boolean` | Flag that sets Product object to active.
+created | `boolean` | The date the Product was generated.
+modified | `boolean` | The date the Product was last updated.
+deleted | `boolean` | Flag that set Product object to deleted.
+company_take |
+name | `string` | The name of the product
+price | `float` | The price of the product
+currency |  `string` | Currency of the product (NOK, EUR, ...) ISO 4217
+business_rules | `list` |
+description | `string` | Product description
+short_description | `string` | Short description about a product
+main_product | `boolean` | To know if a product is main product or not
+_sub_products | `list` | It is a list of sub products under the main product.
+parents | `list` | The sub product parent, overhead product.
+default_position | `geoPoint` | Define Product position.
+max_distance | `int` |
+tags | `list` | List of Tags associated with Product.
+properties | `dict` | Properties is a dictionary where you could add product properties.
+path | `string` | Path is the Product URL for website
+vat | `float` |
+slug | `string` |
+
+## Create a Product
+
+``` http
+POST /products HTTP/1.1
+Content-Type: application/json
+Authorization: Bearer <jwt>
+X-Share-Api-Key: <shareactor-api-key>
+Host: api.shareactor.io
+
+{
+  "product": 
+  {
+        "description": "It is a test product",
+        "currency": "NOK",
+        "price": 3.14,
+        "name": "product_name",
```

Comment:
use proper names "Product Name", otherwise people might think they need to use `_`